blackbox/docs/admin/system-information.rst

```diff
@@ -1042,6 +1042,44 @@ Every request that queries data or manipulates data is considered a "job" if it
 is a valid query. Requests that are not valid queries (for example, a request
 that tries to query a non-existent table) will not show up as jobs.
 
+Jobs Metrics
+------------
+
+The ``sys.jobs_metrics`` table provides an overview of the duration latencies
+of queries executed on the cluster.
+
+.. note::
+
+  The values returned are representive of the overall query execution times but
+  they can have slight in-accurencies in order to reduce the memory requirement
+  of these metrics.
+
+``sys.jobs_metrics`` Table Schema
+.................................
+
+
++------------------------------------+----------------------------------------------------+---------------+
+| Column Name                        | Description                                        |  Return Type  |
++====================================+====================================================+===============+
+| ``node``                           | An object containing node name and id              | ``OBJECT``    |
++------------------------------------+----------------------------------------------------+---------------+
+| ``type``                           | Type of queries that have been taken into account  | ``STRING``    |
+|                                    | for the calculation of the latencies provided.     |               |
++------------------------------------+----------------------------------------------------+---------------+
+| ``total_count``                    | Total number of queries executed                   | ``LONG``      |
++------------------------------------+----------------------------------------------------+---------------+
+| ``mean``                           | Mean duration of a query in ms                     | ``DOUBLE``    |
```

Comment:
Would be good to have a consistent description for `mean`, `min` and `max`, e.g.
"The mean|min|max query duration in ms"

Reply:
Why do we have `DOUBLE` as type here? I guess that's coming from the histogram, but I would make is also `LONG` since we don't measure millisecond parts.

Reply:
Because it's the mean, even with only `long` measurements we can end up having fractional number as mean: `(7ms + 8ms) / 2ms = 7.5ms`

Reply:
That I understand, but I am questioning if we want to display fractional `ms` results. (I'm fine with it)

Reply:
Any reason why we shouldn't?

Reply:
not really. was just an impulsive first thought. nevermind, DOUBLE is totally ok